recipe/meta.yaml

```diff
@@ -96,21 +104,17 @@ requirements:
     # no quesitons about the license of the resulting opencv
     # binary
     - ffmpeg {{ ffmpeg }}=lgpl_*
-    - qt-main                             # [not osx and not ppc64le]
+    - qt-main     # [qt_version == 5]
     - liblapacke
     - liblapack
     - libcblas
     - freetype
-    - glib                           # [unix]
+    - glib
```

Comment:
I don't think that glib is actually required:
https://github.com/conda-forge/conda-forge.github.io/issues/1880